src/dsp_tools/commands/xmlupload/resource_create_client.py

```diff
@@ -41,9 +41,8 @@ def create_resource(
             f"Attempting to create resource {resource.res_id} (label: {resource.label}, iri: {resource.iri})..."
         )
         resource_dict = self._make_resource_with_values(resource, bitstream_information)
-        resource_json_ld = json.dumps(resource_dict, ensure_ascii=False)
         headers = {"X-Asset-Ingested": "true"} if self.media_previously_ingested else None
-        res = self.con.post(route="/v2/resources", jsondata=resource_json_ld, headers=headers)
+        res = self.con.post(route="/v2/resources", jsondata=resource_dict, headers=headers)
```

Comment:
Do we not dump the json here for consistency? There are no PWs...

Reply:
I'm not sure what you mean. Before, every caller of the Connection class first dumped=serialized the dictionary to a JSON-formatted string. But this PR changes that: Now, we pass dictionaries to the Connection class.

So it was my intention to remove line 44 here. This is consistent, because everywhere else, we now do it the same way: passing dicts instead of strings. 

But perhaps I didn't get your point?

Reply:
No I was just curious if consistency was the reason that is all